README.md

```diff
@@ -84,7 +84,12 @@ Tests are an important part of the development and evolution of this project,
 and new contributions are expected to include tests for any functionality
 change.  To run the tests, pass the `test` verb to the `bootstrap` script:
 
-    ./Utilities/bootstrap --build-tests test
+    ./Utilities/bootstrap \
+        --swiftc /path/to/swift-linux-x86_64/bin/swiftc \
```

Comment:
It doesn't have to be a Linux path, no. The path for any of these parameters can be anything, so long as they include a `swiftc`, a `swift-build-tool`, and a Foundation and XCTest built library. I'm fine with any naming scheme here, but I thought this one would be the most intuitive for contributors building on Linux. Let me know if you feel strongly about `path/to/snapshot/usr/bin/swiftc` and I'll change it.

(Aside: The paths need to be absolute, so it'll have to be `/path/to/snapshot/usr/bin/swiftc`. We should modify the `bootstrap` script to either raise an error if given relative paths, or transform relative paths to absolute ones. The Swift build script uses a combination of these two techniques.)